modules/cnf-du-pod-mutation.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-pod-mutation_{context}"]
+
+= Pod mutation
+
+The workload partitioning feature modifies pods that are annotated with `target.workload.openshift.io/management` and that are in a namespace that contains the `workload.openshift.io/allowed: management` annotation.
```

Comment:
In other places we use {workload-type} rather than the specific "management"

Reply:
Change made.